### PR TITLE
fix: added spacing between logos

### PR DESCRIPTION
**Pull request by EfeSenerr**

### CODE REVIEW

File: components/Logos.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export interface Props {
 export default function Logos({ logos }: Props) {
   return (
     <>
-      <div className="flex flex-wrap items-center justify-center">
+      <div className="flex flex-wrap items-center justify-center space-y-4">
```
Comment (simon-hng) [inline components/Logos.tsx:16]: using `gap` with `flex` or `grid` is better as that still applies if you change flex direction for mobile/desktop

**Review by simon-hng — APPROVED** [review]
Really cool that you got all those logos and sponsors. Thanks a lot

Merge it when you think its ready

**Comment by MunzerDw** [comment]
LGTM

**Review by MunzerDw — APPROVED** [review]
Perefect

**Pull request merged**